### PR TITLE
Add Avi Kubernetes Operator (AKO) to the list of implementations

### PR DESCRIPTION
**Pull request by pkoshtavmware**
**What type of PR is this?**
/kind documentation

**What this PR does / why we need it**:
Avi Kubernetes Operator(AKO) now supports Gateway API. This PR adds AKO to the list of implementations

**Which issue(s) this PR fixes**:
Fixes #

**Does this PR introduce a user-facing change?**:
"None"

### CODE REVIEW

**Comment by sunjayBhatia** [comment]
I think you'll need to also add a section to the PR description with a `NONE` release note:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

the markdown code block should begin with "```release-note"

File: site-src/implementations.md [hunk 1]
```diff
@@ -75,6 +76,7 @@ Implementors and integrators of Gateway API are encouraged to update this docume
 [28]:#linkerd
 [29]:#tyk
 [30]:#easegress
+[31]:#ako
```
Comment (frankbu) [inline site-src/implementations.md:79]:
```suggestion
[31]:#avi-kubernetes-operator
```

File: site-src/implementations.md [hunk 2]
```diff
@@ -106,6 +108,18 @@ APISIX currently supports Gateway API `v1beta1` version of the specification for
 [apisix]:https://apisix.apache.org/
 [apisix-1]:https://github.com/apache/apisix-ingress-controller
 
+### Avi Kubernetes Operator
+
+[Avi Kubernetes Operator (AKO)][ako] provides L4-L7 load-balancing using VMware NSX Advanced Load Balancer.
+
+Starting with AKO version [v1.11.1], Gateway API version v0.6.2 is supported. It implements v1beta1 version of Gateway API specification supporting GatewayClass, Gateway and HTTPRoute objects. AKO Gateway API is currently in Tech Preview.
+
+Documentation to deploy and use AKO Gateway API can be found at [Avi Kubernetes Operator Gateway Api][ako-gw]
```
Comment (frankbu) [inline site-src/implementations.md:117]:
```suggestion
Documentation to deploy and use AKO Gateway API can be found at [Avi Kubernetes Operator Gateway Api][ako-gw].
```

File: site-src/implementations.md (hunk 1 above)
Comment (pkoshtavmware) [reply site-src/implementations.md]: Changed

File: site-src/implementations.md (hunk 2 above)
Comment (pkoshtavmware) [reply site-src/implementations.md]: Changed

File: site-src/implementations.md [hunk 3]
```diff
@@ -106,6 +108,18 @@ APISIX currently supports Gateway API `v1beta1` version of the specification for
 [apisix]:https://apisix.apache.org/
 [apisix-1]:https://github.com/apache/apisix-ingress-controller
 
+### Avi Kubernetes Operator
+
+[Avi Kubernetes Operator (AKO)][ako] provides L4-L7 load-balancing using VMware NSX Advanced Load Balancer.
+
+Starting with AKO version [v1.11.1], Gateway API version v0.6.2 is supported. It implements v1beta1 version of Gateway API specification supporting GatewayClass, Gateway and HTTPRoute objects. AKO Gateway API is currently in Tech Preview.
+
+Documentation to deploy and use AKO Gateway API can be found at [Avi Kubernetes Operator Gateway Api][ako-gw].
```
Comment (robscott) [inline site-src/implementations.md:117]:
```suggestion
Documentation to deploy and use AKO Gateway API can be found at [Avi Kubernetes Operator Gateway API][ako-gw].
```

**Review by robscott — COMMENTED** [review]
Thanks @pkoshtavmware!

/ok-to-test
/approve

**Review by Xunzhuo — COMMENTED** [review]
/lgtm

**Pull request merged**